### PR TITLE
Update manage-industry-news.md

### PR DESCRIPTION
**Pull request by msbemba**
Corrected Typos and dupes

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3169

File: microsoft-365/admin/manage/manage-industry-news.md [hunk 1]
```diff
@@ -23,9 +23,7 @@ To provide your users with up-to-date news headlines about your industry and
 
 ## What your users will see
 
-Signed-in users can that come to the Bing homepage will see your industry news feed under the personalized info from your organization.  
-
-Signed-in users can that come to the Bing homepage will see your industry news feed under the personalized info from your organization.
+Signed-in users that come to the Bing homepage will see your industry news feed under the personalized info from your organization.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/manage-industry-news.md]:
```suggestion
Signed-in users who go to the Bing homepage will see your industry's news feed under the personalized info for your organization.  
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@pebaum Please merge this PR. Thanks

**Pull request merged**